hack/all-in-one/README.md

```diff
@@ -68,17 +68,17 @@ kubectl get pods -A
 ### Access the cluster on the host
 #### Linux
 ```bash
-export KUBECONFIG=$(podman volume inspect ushift-vol --format "{{.Mountpoint}}")/microshift/resources/kubeadmin/kubeconfig
+export KUBECONFIG=$(podman volume inspect microshift-data --format "{{.Mountpoint}}")/microshift/resources/kubeadmin/kubeconfig
```

Comment:
can you revert these changes?

Reply:
I will. Can't get the tests to pass though. :thinking:

Reply:
> can you revert these changes?

Will the changes to volume name will go in a different PR or we don't need this change at all? All other places use `microshift-data` as the volume name instead of `ushift-vol`